RELEASE.md

```diff
@@ -27,93 +27,82 @@ liboqs can also be used in the following programming languages via language-spec
 Release notes
 =============
 
-This is version 0.12.0 of liboqs. It was released on December 9, 2024.
+This is version 0.13.0-rc1 of liboqs. It was released on March 19, 2025.
 
-This release updates the ML-DSA implementation to the [final FIPS 204](https://csrc.nist.gov/pubs/fips/204/final) version. This release still includes the NIST Round 3 version of Dilithium for interoperability purposes, but we plan to remove Dilithium Round 3 in a future release.
-
-Deprecation notice
-==================
-
-This will be the last release of liboqs to include Kyber (that is, the NIST Round 3 version of Kyber, prior to its standardization by NIST as ML-KEM in FIPS 203). Applications should switch to ML-KEM (FIPS 203).
-
-The addition of ML-DSA FIPS 204 final version to liboqs has introduced a new signature API which includes a context string parameter. We are planning to remove the old version of the API without a context string in the next release to streamline the API and bring it in line with NIST specifications. Users who have an opinion on this removal are invited to provide input at https://github.com/open-quantum-safe/liboqs/issues/2001.
-
-Security issues
-===============
-
-- CVE-2024-54137: Fixed bug in HQC decapsulation that leads to incorrect shared secret value during decapsulation when called with an invalid ciphertext. Thank you to Célian Glénaz and Dahmun Goudarzi from Quarkslab for identifying the issue.
+This release improves support for NIST Additional Signatures Round 2 candidates: CROSS and MAYO implementations are updated and support is added for UOV. This release also adds a new KEM API for deterministic key generation and encapsulation (only supported by ML-KEM at the moment).
 
 What's New
 ----------
 
-This release continues from the 0.11.0 release of liboqs.
+This release continues from the 0.12.0 release of liboqs.
 
 ### Key encapsulation mechanisms
 
-- HQC: Fixed bug in decapsulation that leads to incorrect shared secret value during decapsulation when called with an invalid ciphertext. Thank you to Célian Glénaz and Dahmun Goudarzi from Quarkslab for identifying the issue.
-- Kyber: This is the last release of liboqs to include Kyber.
-- ML-KEM: Improved testing of ML-KEM.
+- New API: Added a deterministic key generation and encapsulation API for KEMs (only ML-KEM supported at the moment).
+- ML-KEM: Added support for the ML-KEM implementation from [Nvidia cuPQC](https://developer.nvidia.com/cupqc), a GPU accelerated cryptography library.
+- ML-KEM: Implementation from mlkem-native upstream updated to add Pair-wise Consistency Test (PCT) and Intel CET support.
```

Comment:
I believe the mlkem-native implementation is new for this release. May be worth an extra bullet point.